src/index.ts

```diff
@@ -9,6 +9,7 @@ module.exports = async (): Promise<void> => {
     .plugins('./node_modules', { matching: 'solidarity-*', hidden: true })
     // global installs
     .plugins('/usr/local/lib/node_modules', { matching: 'solidarity-*', hidden: true }) // Darwin
+    .plugins(`${__dirname}/../..`, { matching: 'solidarity-*', hidden: true }) // Yarn/Darwin
```

Comment:
This is an interesting solution.  What about this?

https://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way